### PR TITLE
Replace git-rev with git-rev-sync

### PR DESCRIPTION
**Pull request by szkl**
Signed-off-by: Sonmez Kartal <sonmez@koding.com>

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -150,22 +149,22 @@ updateVault = ->
...
-  else
-    version = try (fs.readFileSync path.join(__dirname, './VERSION'), 'utf-8').trim()
-
-  if argv.hostname
-    options.hostname = argv.hostname
-    updateVault()
-  else if os.type() is "Darwin"
-    options.hostname = "dev.koding.com"
+try
+  version = require('git-rev-sync').short()
```
Comment (cihangir) [inline configure]: `version = str.substring 0, 8` is required - there are other places where we apply string expansions

Comment (cihangir) [inline configure]: or we need to fix other places as well :)

File: package.json [hunk 2]
```diff
@@ -121,6 +120,7 @@
     "chai": "3.5.0",
     "coffeelint": "1.16.0",
     "crypto": "0.0.3",
+    "git-rev-sync": "1.8.0",
```
Comment (cihangir) [inline package.json]: 👍

File: configure [hunk 3]
```diff
@@ -150,22 +149,22 @@ updateVault = ->
...
+
+if argv.hostname
+  options.hostname = argv.hostname
+  updateVault()
+else if os.type() is "Darwin"
+  options.hostname = "dev.koding.com"
+  updateVault()
+else
+  log "lemme find your IP"
+  request "https://p.koding.com/-/ip", (err, res, body) ->
```
Comment (cihangir) [inline configure]: also there is https://www.npmjs.com/package/sync-request :)

**Review by cihangir — CHANGES_REQUESTED** [review]
substring

File: configure (hunk 1 above)
Comment (szkl) [inline configure]: I've found no other reference to git-rev module.

**Review by cihangir — APPROVED** [review]

**Pull request merged**